s3sync_test.go

```diff
@@ -34,9 +34,15 @@ func TestS3syncNotImplemented(t *testing.T) {
 	if err := m.Sync("foo", "bar"); err == nil {
 		t.Fatal("local to local sync is not supported")
 	}
+}
+
+func TestS3ToS3(t *testing.T) {
+	m := New(getSession())
 
-	if err := m.Sync("s3://foo", "s3://bar"); err == nil {
-		t.Fatal("s3 to s3 sync is not implemented yet")
+	err := m.Sync("s3://s3-source", "s3://s3-destination")
+
+	if err != nil {
+		t.Fatal(err.Error())
 	}
 }
```

Comment:
Mind adding a check of the sync result like
https://github.com/seqsense/s3sync/blob/baf6e271d9c7fee48bcefe6bbe4b6d7151c6a92b/s3sync_test.go#L163-L181
?

Reply:
Added test in e6b4d8c